PurchasesCoreSwift/LocalReceiptParsing/DataConverters/ArraySlice_UInt8+Extensions.swift

```diff
@@ -32,8 +32,8 @@ extension ArraySlice where Element == UInt8 {
         return String(bytes: self, encoding: .utf8)
     }
 
-    func toDate(dateFormatter: ISO3601DateFormatter) -> Date? {
-        return dateFormatter.date(fromBytes: self)
+    func toDate() -> Date? {
```

Comment:
maybe this one should be named toIso8601Date or something of the sort?

Reply:
yep

Reply:
well, actually, no - this just returns a Date type, it's not specific. It does assume that `self` is a set of bytes in `iso8601` representation of date, so perhaps it could reflect that in the name, although it's not strictly necessar - all other methods here also assume some form of encoding

Reply:
ok fair, makes sense for this class in particular.

Reply:
`toISO8601Date`

Reply:
@taquitos I think that naming might be misleading, given that it returns a generic `Date`. it only assumes that `self` is a slice of bytes in ISO8601 format.

so I'd either go with a generic `toDate`, like other methods in this extension (which also assume format), or come up with a name that represents the fact that the `ISO8601` part is in the _current_ object, not the _returned_ one. 

Hope that makes sense!

Reply:
💯  ignore me